src/components/layout/nav.js

```diff
@@ -178,6 +178,12 @@ const NavLink = styled.li`
 `
 const StyledLink = styled(LocalizedLink)`
     ${SharedLinkStyle}
+
+    &:hover {
```

Comment:
we only want to remove red line on logo, not the entire navbar. tips: use localized link instead of styled link for logo on reset password